components/blog-index/server.js

```diff
@@ -50,8 +51,10 @@ function PostPreview(context, blogMeta) {
         ? html`<span class="sponsored">Sponsored</span>`
         : nothing}
       ${Button.render(context, {
-        label: "Read more →",
+        label: "Read more",
```

Comment:
Nit: We have at least to more cases, so maybe we can fix these together separately?